PGM/src/main/java/tc/oc/pgm/mutation/types/kit/TeamChestMutation.java

```diff
@@ -0,0 +1,135 @@
+package tc.oc.pgm.mutation.types.kit;
+
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.block.Action;
+import org.bukkit.event.inventory.InventoryAction;
+import org.bukkit.event.inventory.InventoryClickEvent;
+import org.bukkit.event.inventory.InventoryType;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.inventory.EquipmentSlot;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import tc.oc.commons.bukkit.inventory.Slot;
+import tc.oc.commons.bukkit.item.ItemBuilder;
+import tc.oc.pgm.PGMTranslations;
+import tc.oc.pgm.kits.ItemKit;
+import tc.oc.pgm.kits.Kit;
+import tc.oc.pgm.kits.SlotItemKit;
+import tc.oc.pgm.match.Match;
+import tc.oc.pgm.match.MatchPlayer;
+import tc.oc.pgm.match.Party;
+import tc.oc.pgm.mutation.types.KitMutation;
+import tc.oc.pgm.wool.WoolMatchModule;
+
+import java.util.*;
+
+public class TeamChestMutation extends KitMutation {
+    final static int SLOT_ID = 17; // Top right
+    final static Material TOOL_TYPE = Material.ENDER_CHEST;
```

Comment:
Do we need this extra field?

Reply:
Because it is used six times, I believe so. So in the future it may be easy to change it if needed.